test/path.js

```diff
@@ -5,7 +5,7 @@ var R = require('..');
 
 describe('path', function() {
   var deepObject = {a: {b: {c: 'c'}}, falseVal: false, nullVal: null, undefinedVal: undefined, arrayVal: ['arr']};
-  it('takes a path and an object and returns the value at the path or undefined', function() {
+  it('takes a path and an object and returns the value at the path or the object', function() {
```

Comment:
This wording change should be reverted. Your argument is that the object *is* the value at the path when the path is `[]`. Also, the function does still return undefined if the path does not exist.